### PR TITLE
Enable use_named_constants_lint

### PR DESCRIPTION
**Pull request by Abhishek01039**
Enable use_named_constants_lint

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by Abhishek01039** [comment]
Hello @a14n 
can you please review this PR?
Thansk

File: packages/flutter/test/gestures/gesture_binding_resample_event_on_widget_test.dart [hunk 1]
```diff
@@ -50,7 +50,7 @@ void main() {
         ui.PointerData(
             change: ui.PointerChange.down,
             physicalX: 0.0,
-            timeStamp: epoch + const Duration(milliseconds: 0),
+            timeStamp: epoch + Duration.zero,
```
Comment (a14n) [inline packages/flutter/test/gestures/gesture_binding_resample_event_on_widget_test.dart:53]: Here and at L48 I would remove `+ Duration.zero`

File: packages/flutter/test/gestures/gesture_binding_resample_event_test.dart [hunk 2]
```diff
@@ -69,7 +69,7 @@ void main() {
         ui.PointerData(
             change: ui.PointerChange.add,
             physicalX: 0.0,
-            timeStamp: epoch + const Duration(milliseconds: 0),
+            timeStamp: epoch + Duration.zero,
```
Comment (a14n) [inline packages/flutter/test/gestures/gesture_binding_resample_event_test.dart:72]: ditto

**Review by a14n — APPROVED** [review]
LGTM

File: packages/flutter/test/gestures/gesture_binding_resample_event_on_widget_test.dart (hunk 1 above)
Comment (Abhishek01039) [reply packages/flutter/test/gestures/gesture_binding_resample_event_on_widget_test.dart]: Great thanks @a14n

**Review by a14n — APPROVED** [review]
LGTM

**Pull request merged**